### PR TITLE
Issue #4 - set up react-router: done, add basic router, navigation co…

### PR DESCRIPTION
**Pull request by myroslav-kolomiiets**
Issue #4 - set up react-router: done, add basic router, navigation component and basic templates to test if it works

### CODE REVIEW

File: src/components/MoviePage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import logo from '../logo.svg';
+import '../App.css';
+
+class MoviePage extends Component {
+  render() {
+    return (
+      <div>
```
Comment (zonzujiro) [inline src/components/MoviePage.js]: You can use `React.Fragment` or shortcut for this `<>`

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import ActorPage from './ActorPage';
+import MoviePage from './MoviePage';
+import MoviesList from './MoviesList';
+
+class Navigation extends Component {
+  render() {
+    return (
+      <div className="App">
+        <Router>
+          <div>
```
Comment (zonzujiro) [inline src/components/Navigation.js]: Do you need this `div` here?

File: .eslintrc [hunk 3]
```diff
@@ -13,4 +13,4 @@
     "strict": ["error", "global"],
     "curly": "warn"
   }
-}
+}
```
Comment (zonzujiro) [inline .eslintrc]: You have remove last line in file? What for?

**Review by zonzujiro — CHANGES_REQUESTED** [review]
LGTM, but see comments

File: src/components/Navigation.js (hunk 2 above)
Comment (myroslav-kolomiiets) [reply src/components/Navigation.js]: It seems necessary. Without this <div> wrapper - react fails with message 

> Invariant Violation: A Router may have only one child element

Comment (myroslav-kolomiiets) [reply src/components/Navigation.js]: But I replaced it with Fragment and everything is fine to.

**Comment by myroslav-kolomiiets** [comment]
@zonzujiro thank you for comments, all changes are implemented.

File: src/components/MoviePage.js (hunk 1 above)
Comment (myroslav-kolomiiets) [reply src/components/MoviePage.js]: Done

File: .eslintrc (hunk 3 above)
Comment (myroslav-kolomiiets) [reply .eslintrc]: It was removed by mistake, I turned it back,

**Review by zonzujiro — APPROVED** [review]

**Comment by zonzujiro** [comment]
:fire:

**Pull request merged**